api/Helpers/Extensions/ClaimsPrincipalExtensions.cs

```diff
@@ -102,5 +102,12 @@ public static List<string> Roles(this ClaimsPrincipal claimsPrincipal) =>
 
         public static string UserId(this ClaimsPrincipal claimsPrincipal) =>
             claimsPrincipal.FindFirstValue(CustomClaimTypes.UserId);
+
+        public static int JudgeId(this ClaimsPrincipal claimsPrincipal)
+        {
+            var value = claimsPrincipal.FindFirstValue(CustomClaimTypes.JudgeId);
+
+            return int.TryParse(value, out var userId) ? userId : default;
+        }
```

Comment:
This would feel much more natural as an extension property, but it appears we have to wait for .net Core 10 and C#14 for that; https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/extension-methods#layer-specific-functionality